spec/requests/hearings_spec.rb

```diff
@@ -1,53 +1,83 @@
 # frozen_string_literal: true
 
-require 'court_data_adaptor'
-
-RSpec.describe 'hearings', type: :request do
+RSpec.describe 'hearings_v2', type: :request do
   let(:user) { create(:user) }
-  let(:hearing_fixture) { load_json_stub('hearing_by_id_body.json') }
-  let(:hearing_id_from_fixture) { '345be88a-31cf-4a30-9de3-da98e973367e' }
+  let(:hearing_id) { '345be88a-31cf-4a30-9de3-da98e973367e' }
   let(:case_reference) { 'TEST12345' }
-  let(:prosecution_case_fixture) { load_json_stub('unlinked/prosecution_case_by_reference_body.json') }
-
-  before do
-    stub_request(:get, %r{#{api_url}/prosecution_cases.*})
-      .to_return(
-        body: prosecution_case_fixture,
-        headers: { 'Content-Type' => 'application/vnd.api+json' }
-      )
-  end
 
-  context 'when authenticated' do
+  context 'when authenticated', :stub_v2_hearing_summary, :stub_v2_hearing_data, :stub_v2_hearing_events,
+          :stub_case_search do
     before do
       sign_in user
-      get "/hearings/#{hearing_id_from_fixture}?page=0&urn=#{case_reference}"
+      get "/hearings/#{hearing_id}?page=0&urn=#{case_reference}"
     end
 
-    it { expect(response).to render_template('hearings/show') }
+    context 'when v2 is enabled' do
```

Comment:
do we still need the context `when v2 is enabled` ?

Reply:
Nope! I have removed it.